tests/integration/targets/test_mysql_user/tasks/test_user_password_update.yml

```diff
@@ -0,0 +1,178 @@
+# test code update password for the mysql_user module
+# (c) 2014,  Wayne Rosario <wrosario@ansible.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 dof the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+- vars:
+    mysql_parameters: &mysql_params
+      login_user: '{{ mysql_user }}'
+      login_password: '{{ mysql_password }}'
+      login_host: 127.0.0.1
+      login_port: '{{ mysql_primary_port }}'
+
+  block:
+
+    # ============================================================
+    # Update user password for a user.
+    # Assert the user password is updated and old password can no longer be used.
+    #
+    - name: create user1 state=present with a password
+      mysql_user:
+        <<: *mysql_params
+        name: '{{ user_name_1 }}'
+        password: '{{ user_password_1 }}'
+        priv: '*.*:ALL'
```

Comment:
@bmildren Do you know why we're granting ALL privileges here? It doesn't seem like we need that for these tests and it's causing the tests to fail because of #77

Reply:
Don't wait the responce to much, it's quite inefficient  at least for the last 6 months.
I think Ben probably doesn't know the answer anyway (if you look at git blame and see Ben there- he moved the stuff from community.general to this repo)

Reply:
> I think Ben probably doesn't know the answer anyway (if you look at git blame and see Ben there- he moved the stuff from community.general to this repo)

That already kind of gives me the answer I needed. My guess is this particular set of tests were moved here from community.general where they had already been forgotten. I re-added them on this PR as per your comment and they got hit by issue #77.

I do not think we need the privileges option for this particular battery of tests to verify what it's supposed to check so I'm going to go ahead and remove those lines. Then I'll see if there are any further failing tests and what can be done about them if so.

@Andersson007 I'll request your review again once the test dance is done.

Reply:
@Jorge-Rodriguez sounds good, thanks!